### PR TITLE
refactor: consolidate price delta components

### PR DESCRIPTION
**Pull request by cartcrom**
## Description

Updates all areas in the app that have a green/red delta arrow to use the same `<PriceDelta delta={number} />` component
* Moves existing delta component / utils out of the price chart file
* gets rid of overly-nested divs (multiple `ArrowCell` components)
* Replaces PortfolioArrow component with DeltaArrow

### Token Details
|    Before    |   After    |
| ------------ | ------------ |
| image | image |

### Token Explore
|    Before    |   After   |
| ------------ | ----------- |
| image  | image |

### Mini Portfolio
|    Before    |   After   |
| ------------ | ----------- |
| image  | image |

### NFT Collection Details
|    Before    |   After   |
| ------------ | ----------- |
| image | image |

### NFT Explore Table
|    Before    |   After   |
| ------------ | ----------- |
| image | image |

### Search
|    Before    |   After   |
| ------------ | ----------- |
| image | image |

### CODE REVIEW

File: src/components/AccountDrawer/MiniPortfolio/Tokens/index.tsx [hunk 1]
```diff
@@ -115,7 +114,7 @@ function TokenRow({ token, quantity, denominatedValue, tokenProjectMarket }: Tok
                 })}
               </ThemedText.SubHeader>
               <Row justify="flex-end">
-                <PortfolioArrow change={percentChange} size={20} strokeWidth={1.75} />
```
Comment (cartcrom) [inline src/components/AccountDrawer/MiniPortfolio/Tokens/index.tsx:118]: the inline styling here wasn't actually doing anything

**Review by cbachmeier — APPROVED** [review]
LGTM

**Comment by cbachmeier** [comment]
@infredible @bryankuuu @callil 

I noticed a few spots where we have inconsistent gaps between the delta and arrow, as well as one case where the arrow comes after the delta. Do we want to standardize this? Most places in the app have no gap and the arrow comes before the delta.
Search result: 2px gap
Screenshot 2023-09-07 at 9 14 54 AM
TDP: arrow after delta
Screenshot 2023-09-07 at 9 18 48 AM
Token Explore Desktop: 3px gap
Screenshot 2023-09-07 at 9 22 15 AM
Token Explore Mobile: 3px gap
Screenshot 2023-09-07 at 9 23 05 AM

**Review by JackShort — APPROVED** [review]

File: src/components/Tokens/TokenTable/TokenRow.tsx [hunk 2]
```diff
@@ -103,8 +103,9 @@ const StyledTokenRow = styled.div<{
   }
 `
 
-const ClickableContent = styled.div`
+const ClickableContent = styled.div<{ gap?: string }>`
   display: flex;
+  ${({ gap }) => gap && `gap: ${gap}`};
```
Comment (natew) [inline src/components/Tokens/TokenTable/TokenRow.tsx:108]: if this is only ever used for px i think `number` would be better

**Review by natew — APPROVED** [review]

**Review by cbachmeier — APPROVED** [review]

**Pull request merged**